crates/cairo-lint-core/src/lints/manual/manual_unwrap_or_default.rs

```diff
@@ -0,0 +1,166 @@
+use cairo_lang_semantic::{db::SemanticGroup, Condition, ExprBlock, FixedSizeArrayItems, Statement};
+use cairo_lang_defs::plugin::PluginDiagnostic;
+use cairo_lang_diagnostics::Severity;
+use cairo_lang_semantic::{Expr, Arenas};
+use cairo_lang_syntax::node::{TypedStablePtr, TypedSyntaxNode};
+use num_bigint::BigInt;
+
+pub const MANUAL_UNWRAP_OR_DEFAULT: &str = "This can be done in one call with `.unwrap_or_default()`";
+
+pub const DEFAULT: &str = "\"Default::default\"";
+pub const DEFAULT_APPEND: &str = "Default::default";
```

Comment:
what's the diffference between the 2 ?

Reply:
I'm thinking the First might not be useful, as the Default::default is appended to the type.
So i'll proceed to removing the first.

Reply:
Updated